### ISSUE TITLE
"Today" tag removed when 

### ISSUE THREAD

**Issue by stefanopagliari** [ksalzke/scheduling-omnifocus-plugin#7]
Thank you for this plugin.  It is very helpful.
I have a couple of questions about the behaviour of the "today" tag, because it is currently not working as I thought it would, but maybe this is by design.

I'm running OF4 beta, and these are my settings:
- in the plugin setting, I've specified a "Schedule" tag and a "Today" tag
-  in the forecast view I have added "Today" as the tag in the setting "today includes".

ISSUE 1: RESCHEDULING TO TODAY
- I would like to be able to pick items and add them to my "today" view in the forecast view by selecting "reschedule task" and scheduling for today. When I do that, however, no "today" tag or any tag is added to the task (and as a result the task does not appear in the forecast view for today). Selecting a separate date instead leads to the tag to be added. Also selecting "Flag denotes today task" ends up in the task being flagged.

ISSUE 2: DISAPPEARING TODAY TAG
- When I run "Update Schedule", the "today" tag disappear if there are no task schedule for that day. As a result, from the preferences of the plugin the value for "Today tag" becomes none.

Are these two behaviours expected? I'm unclear on whether I'm using the "today" tag correctly

Thank you!

**Comment by ksalzke** [comment]
Hi @stefanopagliari! The way you expect things to work is indeed the way they should work.

I did some quick testing and I suspect you may have created the "Today" as a nested tag within the "Schedule" Tag? 

At least for now, it needs to be outside of that tag. If it's nested inside, could you try moving it out and see whether the plug-in behaviour is more in line with what you expect?

If not, we can do some more troubleshooting.

**Comment by stefanopagliari** [comment]
Hi @ksalzke. You were correct. I had created the "Today" as a nested tag within the "Schedule" Tag. I've now moved that task outside of the Schedule tag and it all works as expected. Thank you for your help

Just a curiosity. Is there any way to see other scheduled tasks in the forecast view (similarly to Things 3)? The settings of the forecast view includes only the option to identifying the task for "today", but I was wondering if you had come across any other workaround.
